docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 mysql:
-    image: autopilotpattern/mysql:latest
+    image: autopilotpattern/mysql
```

Comment:
Does this actually affect the tests, or are you eliminating the explicit `:latest`, since it's assumed anyway?